schemas/oic.types-schema.json

```diff
@@ -32,6 +32,12 @@
           "description": "A string instance is valid against this attribute if it contains an integer in the range [0, (2**64)-1]",
           "type": "string",
           "pattern": "^0|([1-9][0-9]{0,19})$"
+        },
+        "locn-room-list": {
+          "enum": ["unknown","attic","balcony","ballroom","bathroom","bedroom","border","boxroom","cellar","cloakroom","conservatory","corridor","deck","den","diningroom","drawingroom","driveway","dungeon","ensuite","entrance","familyroom","garage","garden","guestroom","hall","kitchen","larder","lawn","library","livingroom","lounge","mancave","masterbedroom","musicroom","office","pantry","parkinglot","parlour","patio","receiptionroom","restroom","roof","roofterrace","sauna","shed","sittingroom","snug","spa","studio","suite","swimmingpool","terrace","toilet","utilityroom","vegetableplot","ward","yard"]
```

Comment:
Do we have to include "type": "string" for locn-room-list and pos-descriptions?  All of the enums are strings, of course, but I'm not sure if you also have to define it as a "string" at the same time.

Note that all of the other Properties in this schema include a "type" parameter.